specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -1117,7 +1117,7 @@
       "properties": {
         "name": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
```

Comment:
May I ask whether here "name" property under "latestAssignmentReport" property is the same thing with the "[name](https://github.com/Azure/azure-rest-api-specs/blob/4bab0bfdf7b1e2e3a50eeda420c2891d087d1d85/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L44)" property in parameters property?